crates/oxc_parser/examples/parser.rs

```diff
@@ -12,7 +12,7 @@ use oxc_parser::Parser;
 fn main() {
     let name = env::args().nth(1).unwrap_or_else(|| "test.js".to_string());
     let path = Path::new(&name);
-    let source_text = std::fs::read_to_string(path).expect("{name} not found");
+    let source_text = std::fs::read_to_string(path).expect(&format!("{name} not found"));
```

Comment:
Noticed this `.expect()` call does not properly interpolate without using the `format!()` macro